plugins/modules/database/influxdb/influxdb_retention_policy.py

```diff
@@ -140,6 +204,21 @@ def create_retention_policy(module, client):
     default = module.params['default']
     shard_group_duration = module.params['shard_group_duration']
 
+    if not check_duration_literal(duration):
+        module.fail_json(msg="Failed to parse value of duration")
+
+    influxdb_duration_format = parse_duration_literal(duration)
+    if influxdb_duration_format != 0 and influxdb_duration_format < 3600000000000:
```

Comment:
That `DURATION_MULTIPLIER` could (if better named :-) ) be used to get rid of those `000000`